src/items/hit-location-item/hitLocationSheet.ts

```diff
@@ -312,28 +319,27 @@ export class HitLocationSheet extends RqgItemSheet<
       return;
     }
 
-    const previousEffect = health2Effect.get(actorHealthBefore);
     const newEffect = health2Effect.get(token.actor.data.data.attributes.health);
 
-    if (newEffect?.label !== previousEffect?.label) {
-      const asOverlay = newEffect?.id === "dead";
-      const newEffectIsOn = !!token.actor.effects.find(
-        (e: ActiveEffect) => e.getFlag("core", "statusId") === newEffect?.id
-      );
-      const previousEffectIsOn = !!token.actor.effects.find(
-        (e: ActiveEffect) => e.getFlag("core", "statusId") === previousEffect?.id
+    for (const status of woundStatuses) {
```

Comment:
the constant woundStatuses is redundant since it's already defined in health2Effect
```suggestion
    for (const status of health2Effect.values()) {
```

Reply:
Done.  I knew it was there but couldn't figure out how to get to it!